pontoon/sync/core/translations_to_repo.py

```diff
@@ -1,35 +1,69 @@
 import logging
 
 from collections import defaultdict
-from collections.abc import Container
 from datetime import datetime
+from itertools import groupby
 from os import remove
 from os.path import commonpath, isfile, join, normpath
 
+from moz.l10n.formats import Format
+from moz.l10n.model import (
+    Entry,
+    Id,
+    Metadata,
+    PatternMessage,
+    Resource,
+    Section,
+    SelectMessage,
+)
 from moz.l10n.paths import L10nConfigPaths, L10nDiscoverPaths
+from moz.l10n.resource import parse_resource, serialize_resource
 
 from django.conf import settings
 from django.db.models import Q
-from django.db.models.manager import BaseManager
+from django.db.models.query import QuerySet
 
 from pontoon.base.models import Locale, Project, Translation, User
 from pontoon.base.models.changed_entity_locale import ChangedEntityLocale
 from pontoon.sync.core.checkout import Checkouts
-from pontoon.sync.formats import parse
-from pontoon.sync.formats.po import POResource
 from pontoon.sync.repositories import CommitToRepositoryException, get_repo
 
 
 log = logging.getLogger(__name__)
 
+# Retaining these in the .po files is unnecessary & misleading,
+# and changes to POT-Creation-Date cause unnecessary churn.
+# Pontoon itself is the appropriate reference.
+gettext_trim_headers = (
+    "Language-Team",
+    "Last-Translator",
+    "PO-Revision-Date",
+    "POT-Creation-Date",
+    "Report-Msgid-Bugs-To",
+    "X-Generator",
+)
+
+# Hacky solution for https://github.com/mozilla-mobile/firefox-ios/issues/9632
+# from https://github.com/mozilla-l10n/firefoxios-l10n/blob/d60eef5ae23fde3f5bcd6d8e5290aab5fd5cc282/.github/scripts/update_other_locales.py#L130-L139
+# TODO: This should almost certainly be handled better
+ios_locale_map = {
+    "ga-IE": "ga",
+    "nb-NO": "nb",
+    "nn-NO": "nn",
+    "sat": "sat-Olck",
+    "sv-SE": "sv",
+    "tl": "fil",
+    "zgh": "tzm",
+}
```

Comment:
Same question as above. As demonstrated by the links in the comment, this isn't Pontoon-specific behavior.

Reply:
It's not Pontoon-specific, but it's really hacky, and smells way too much like cargo cult programming to me. If we were to include this in moz.l10n, removing it would be a breaking change. Dropping it later from Pontoon would not be.